src/lib/store.ts

```diff
@@ -44,6 +44,7 @@ export interface Settings {
 	ollamaServer: string | null;
 	ollamaModel: string | null;
 	ollamaModels: OllamaModel[];
+	userTheme: 'light' | 'dark' | undefined;
```

Comment:
I just noticed that here you can do:
```suggestion
	userTheme?: 'light' | 'dark';
```